openwisp_monitoring/monitoring/tests/__init__.py

```diff
@@ -25,11 +126,15 @@ class TestMonitoringMixin(TestOrganizationMixin):
     def setUpClass(cls):
         setattr(app_settings, 'INFLUXDB_DATABASE', cls.TEST_DB)
         create_database()
+        for key, value in test_charts.items():
+            register_chart(key, value)
 
     @classmethod
     def tearDownClass(cls):
         get_db().drop_database(cls.TEST_DB)
         setattr(app_settings, 'INFLUXDB_DATABASE', cls.ORIGINAL_DB)
+        for key in test_charts.keys():
+            unregister_chart(key)
```

Comment:
Can you check if tests pass without this piece of code? I think this is not needed